Source/EasyNetQ.Tests/ConsumeTests/When_an_error_occurs_in_the_message_handler.cs

```diff
@@ -14,11 +14,10 @@ public class When_an_error_occurs_in_the_message_handler : ConsumerTestBase
 
         protected override void AdditionalSetUp()
         {
-            ConsumerErrorStrategy.Expect(x => x.HandleConsumerError(null, null))
-                     .IgnoreArguments()
-                     .Return(AckStrategies.Ack);
+            ConsumerErrorStrategy.HandleConsumerError(null, null)
+                     .ReturnsForAnyArgs(AckStrategies.Ack);
 
-            exception = new ApplicationException("I've had a bad day :(");
+            exception = new Exception("I've had a bad day :(");
```

Comment:
I'd suggest creating a new exception (ie. EasyNetQTestException), so you can verify that the right kind of exception was thrown and caught.